package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@lokalise/eslint-plugin",
+  "version": "1.0.0",
+  "description": "ESLint plugin for Lokalise",
+  "main": "index.js",
+  "author": "Lokalise",
+  "license": "MIT",
+  "files": [
+    "rules",
+    "index.js",
+    "package.json",
+    "README.md",
+    "LICENSE"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/lokalise/eslint-plugin.git"
+  },
+  "keywords": [
+    "eslint",
+    "imports",
+    "boundaries",
+    "javascript"
+  ],
+  "scripts": {
+    "test": "jest"
+  },
+  "devDependencies": {
```

Comment:
are you sure this has no runtime dependencies? I see imports from eslint.
are there any peerDependencies?

Reply:
Adjusted. Thank you. Let me know.